docs/adding-builtins.md

```diff
@@ -1,53 +0,0 @@
-# Adding Built-in Functions
-
```

Comment:
There's an example that will be shown in the GoDoc now so this file is no longer necessary.